model_service/models/llama2/model.py

```diff
@@ -215,49 +249,22 @@ def generate(self, request):
         results = response_object.json()
 
         # Compile the results into a structure consistent with other kaleidoscope models
-        activations = []
-        generated_sequences = []
+        activations = results["choices"][0]["activations"]
+        generated_sequences = GENERATOR.tokenizer.decode(results["choices"][0]["text"][0])
         tokens = []
-        logprobs = []
-        logger.info(f"{results}")
-        for result in results["choices"][0]["text"]:
-            #activations.append(result["activations"])
-            #activations.append(torch.empty(0))
-            generated_sequences.append(result)
-            #tokens.append(torch.empty(0))
-            #logprobs.append(torch.empty(0, dtype=bytes))
+        for sequence in results["choices"][0]["text"]:
```

Comment:
```suggestion
        for sequence in results["choices"][0]["text"][0]:
```
Based on https://github.com/VectorInstitute/kaleidoscope/pull/68/files#r1302074740, `[0]` should be added here as well right?

Reply:
Nice catch! It turns out the response generation changed after we fixed a different bug, and the `[0]` should not have been here in the first place. I've removed it.